.github/workflows/generate.yml

```diff
@@ -23,6 +23,8 @@ jobs:
 
       - name: Test static generation
         run: |
+          curl -LO https://dl.k8s.io/release/v1.21.0/bin/linux/amd64/kubectl
+          sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
```

Comment:
I don't see any added dependency on kubectl in this PR. Why is this part needed?

Reply:
Same as the above reason.